packages/components/src/link/test/index.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * External dependencies
+ */
+import { fireEvent, render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import Link from '../index';
+
+describe( 'Link', () => {
+	it( 'should render `external` links', () => {
+		const { container } = render(
+			<Link href="https://woocommerce.com" type="external">
+				WooCommerce.com
+			</Link>
+		);
+
+		expect( container.firstChild ).toMatchInlineSnapshot( `
+			<a
+			  data-link-type="external"
+			  href="https://woocommerce.com"
+			>
+			  WooCommerce.com
+			</a>
+		` );
+	} );
+
+	it( 'should render `wp-admin` links', () => {
+		const { container } = render(
+			<Link href="post-new.php?post_type=product" type="wp-admin">
+				New Post
+			</Link>
+		);
+
+		expect( container.firstChild ).toMatchInlineSnapshot( `
+			<a
+			  data-link-type="wp-admin"
+			  href="post-new.php?post_type=product"
+			>
+			  New Post
+			</a>
+		` );
+	} );
+
+	it( 'should render `wc-admin` links', () => {
+		const { container } = render(
+			<Link
+				href="admin.php?page=wc-admin&path=%2Fanalytics%2Forders"
+				type="wc-admin"
+			>
+				Analytics: Orders
+			</Link>
+		);
+
+		expect( container.firstChild ).toMatchInlineSnapshot( `
+			<a
+			  data-link-type="wc-admin"
+			  href="admin.php?page=wc-admin&path=%2Fanalytics%2Forders"
```

Comment:
`Link` actually doesn't do anything with the href passed in. It is up to the caller of `Link` to construct the href appropriately. It could be a good enhancement to make to `Link` at some point, though. I'll backlog that idea.